pwnlib/elf/elf.py

```diff
@@ -1328,13 +1328,23 @@ def relro(self):
         .. _DF_BIND_NOW: http://refspecs.linuxbase.org/elf/gabi4+/ch5.dynamic.html#df_bind_now
 
         """
+        if not any('GNU_RELRO' in str(s.header.p_type) for s in self.segments):
```

Comment:
Alternately, we could just add tests which generate the binaries on-the-fly by shelling out to GCC.  This would keep the Git repo size down, and effectively add to the documentation about the different flags / states.